src/views/home/components/amount-input/amount-input.view.tsx

```diff
@@ -15,25 +15,28 @@ interface AmountInputProps {
   value?: BigNumber;
   token: Token;
   balance: BigNumber;
-  fee: BigNumber;
+  fee?: BigNumber;
   onChange: (params: onChangeParams) => void;
 }
 
 const AmountInput: FC<AmountInputProps> = ({ value, token, balance, fee, onChange }) => {
   const defaultInputValue = value ? formatUnits(value, token.decimals) : "";
   const [inputValue, setInputValue] = useState(defaultInputValue);
+  const [actualFee, setActualFee] = useState<BigNumber>();
```

Comment:
We should initialize this as zero since later, in the effect, when `fee` is `undefined` we set it as zero.

Setting zero as the initial value we get rid of the `undefined` case and avoid having to initialize it asynchronously later in the effect.